components/bl-badge-component/component.json

```diff
@@ -0,0 +1,131 @@
+{
+  "id": "c_7e8237be9ed8a45d5a5e99d7c1ffd257",
+  "name": "Badge",
+  "description": "Badge generates a small badge to the corner of its child(ren).",
+  "showInToolbox": true,
+  "faIcon": "certificate",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "name": "badgeLabel",
+      "label": "Badge Label",
+      "type": "text",
+      "handlerId": "badgeLabelLogic",
+      "handlerLabel": "Badge Label Logic",
+      "handlerDescription": "This is a handler for the logic to determine the label of the Badge. Value returned by the handler will be used as the label of the Badge.",
+      "dataBinding": true
+    },
+    {
+      "name": "badgeLabelColor",
+      "label": "Badge Label Color",
+      "type": "text"
+    },
+    {
+      "name": "badgeBackgroundColor",
+      "label": "Badge Background Color",
+      "type": "text"
+    },
+    {
+      "name": "badgeFontSize",
+      "label": "Badge Font Size",
+      "type": "number"
```

Comment:
Is it still number? I believe after our disscussion about rem/em/% there should be a `text` type